### PR TITLE
README.md update for Linux users

### PR DESCRIPTION
**Pull request by rfontoura**
Added step-by-step section with installation instructions for Linux users that perhaps doesn't have some dependencies installed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+
+1. Make sure you have `zsh` installed on your machine (default location is `/bin/zsh`). Type `zsh --version` to see if you have it. If not, install it running the command `sudo apt-get install zsh`
+1. Make sure you have `vim` installed. If not, install it with `sudo apt-get install vim`
+1. Run the installation script running ```sh -c "`curl -fsSL https://raw.githubusercontent.com/lfilho/dotfiles/master/install.sh`"```
+1. If asked, provide your superuser password
+1. Configure your terminal program to point the command to zsh (default `/bin/zsh`)
+1. Restart you Terminal
+1. You're done.
+
+# Install (for another Users)
```
Comment (lfilho) [inline README.md]: Could this be `for OSX users` instead? As it won't work for Windows, for instance.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 **This is @lfilho's version of YADR.**
 
-# Install
+# Install (for Linux Users)
+
+1. Make sure you have `zsh` installed on your machine (default location is `/bin/zsh`). Type `zsh --version` to see if you have it. If not, install it running the command `sudo apt-get install zsh`
```
Comment (lfilho) [inline README.md]: Although this Markdown numbering works beautifully in the browser, it doesn't look as good for people looking at the .md file directly in a file editor, for example (original goal of creating documents in markdown - to make it look good and simple even as plain text). So I think it's always a good practice to make it read 1., 2., 3., etc.

But of course that's not a big deal. Feel free to only do that if you have the time 😄

File: README.md (hunk 1 above)
Comment (rfontoura) [inline README.md]: I tought about another Unix users... so, change it to `for OSX users`?

File: README.md (hunk 2 above)
Comment (rfontoura) [inline README.md]: OK, I'll do this changes.

**Comment by rfontoura** [comment]
Requested changes done.

**Comment by lfilho** [comment]
Thanks, man! Merging in 3, 2, 1...

**Pull request merged**